servicex/query.py

```diff
@@ -157,9 +156,10 @@ def set_result_format(self, result_format: ResultFormat):
         return self
 
     async def submit_and_download(
-        self, signed_urls_only: bool,
```

Comment:
Formatting changes in the same commit as the code changes make the review a lot harder. If you want to change formatting, it's more polite to make that a single commit and then have the meat of the PR in a second commit